keylime/config.py

```diff
@@ -295,6 +295,10 @@ def valid_regex(regex):
 
 IMA_PCR = 10
 
+# measured boot addons
+MEASUREDBOOT_PCRS = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 14]
```

Comment:
not a blocker by any means, but I wonder if this might be useful exposed in keylime.conf. I guess not and you would know better here.

Reply:
I thought about it, and decided to keep all values "hard-coded" mostly because - very much like IMA - "measured boot" refers to a very specific set of PCRs. Granted, there is some latitude on PCR 14 (which is updated only when secure booting), but I plan to revisit it after the basic set (of 3) PRs gets merged.